app/views/devise/sessions/new.html.erb

```diff
@@ -1,26 +1,34 @@
-<h2>Log in</h2>
+<%= stylesheet_link_tag "signin_form" %>
```

Comment:
This style guide will be included in the middle of the HTML, that is not quite right, check out the `content_for` method for Rails to include this on the header dinamically